packages/shared/src/components/modals/squads/NewSquadModal.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { ReactElement, useCallback } from 'react';
+import classNames from 'classnames';
+import { Modal, ModalProps } from '../common/Modal';
+import { ModalSize } from '../common/types';
+import { Button, ButtonVariant } from '../../buttons/Button';
+import { useSquadNavigation } from '../../../hooks';
+import { webappUrl } from '../../../lib/constants';
+import { useSlack } from '../../../hooks/integrations/slack/useSlack';
+import { Image } from '../../image/Image';
+import { cloudinary } from '../../../lib/image';
+import {
+  Typography,
+  TypographyColor,
+  TypographyType,
+} from '../../typography/Typography';
+import { SlackIcon } from '../../icons';
+import OrDivider from '../../auth/OrDivider';
+import { ModalClose } from '../common/ModalClose';
+import { LogEvent, Origin } from '../../../lib/log';
+import { UserIntegrationType } from '../../../graphql/integrations';
+import { useLogContext } from '../../../contexts/LogContext';
+
+export default function NewSquadModal(props: ModalProps): ReactElement {
+  const modalProps: ModalProps = {
+    ...props,
+  };
```

Comment:
Do we still need this?

Reply:
Ah it's mostly copy paste issue, I'll change.

Nice find!